docs/how-to/copier_update.md

```diff
@@ -35,7 +35,7 @@ You can supply the VERSION_NUMBER of the template you want or omit the `-r` opti
 
 This will update your project in place. You should then inspect the changes using git (the source control pane in vscode is excellent for this purpose) and commit them to your repository. Once you are happy with the changes you can test them by re-deploying some of your IOC instances. When everything is working you can push the changes to your repository.
 
-The template comes with an example IOC called xxxx-ea-test-01. You are free to delete this if you don't want it. However, we recommend keeping that IOC as it is a good reference for what changes might be needed in your own IOC instances.
+The template comes with an example IOC called `.ioc_template`. You are free to delete this if you don't want it. However, we recommend keeping that IOC as it is a good reference for what changes might be needed in your own IOC instances.
```

Comment:
This text should be changed because it is no longer a functioning example, rather a template.

Reply:
How does 7000d01f24ee9a87109233a5d27c0573da81e00e sound?

Reply:
How does 73c1af285a6ff108f0f05a815e1dabea14d78ee2 sound?

Reply:
In fact I think this paragraph could be deleted as the use of .ioc_template is described elsewhere.

Reply:
Deleted in 2419be8e2f21549aac0ae0721b313d034cb47661

Reply:
Yay - deleting the paragraph is better. Deleting the .ioc_template is not a good idea as it will track changes to what services folders for IOCs should look like through changes in the framework.

Thanks and merging!